src/tests/suites/RedactUtility-Test.ts

```diff
@@ -0,0 +1,24 @@
+import * as assert from 'assert';
+import { redactJsonObject } from '../../utility/Redact';
+
+suite('Redact Test', () => {
+    test('should return redacted secrets always', async () => {
```

Comment:
Can we add a UT that covers the case where invalid JSON is passed? I want to make sure we wont crash in such case.